### PR TITLE
Skip empty repositories for Allstar policy enforcement

### PR DESCRIPTION
**Pull request by raghavkaul**
Skip empty repositories for Allstar policy enforcement. Check whether they're empty using [GitHub REST API](https://docs.github.com/en/rest/metrics/statistics?apiVersion=2022-11-28#get-all-contributor-commit-activity).

### CODE REVIEW

**Comment by jeffmendoza** [comment]
Looking at https://docs.github.com/en/rest/metrics/statistics?apiVersion=2022-11-28#best-practices-for-caching and https://pkg.go.dev/github.com/google/go-github/v56/github#RepositoriesService.ListContributorsStats This doesn't seem like a good option to use. It will likely result in an error/202 and we don't need this expensive operation to tell if the repo is empty.

Is using something like https://pkg.go.dev/github.com/google/go-github/v56/github#RepositoriesService.GetContents an option and seeing if the top-level is empty?

**Comment by raghavkaul** [comment]
Yes, that API makes sense. Updated!

File: pkg/enforce/enforce.go [hunk 1]
```diff
@@ -318,6 +321,24 @@ func EnforceJob(ctx context.Context, ghc *ghclients.GHClients, d time.Duration,
...
+	}
+	if file == nil && len(dirs) == 0 {
+		log.Info().
+			Str("org", owner).
+			Str("repo", repo).
+			Str("area", policy).
+			Msg("Repository is empty, skipping")
+	}
+
+	return true, nil
```
Comment (jeffmendoza) [inline pkg/enforce/enforce.go]: Probably want to return false here, and true inside the if statement.

Comment (raghavkaul) [reply pkg/enforce/enforce.go]: Essential catch!

**Review by jeffmendoza — APPROVED** [review]

**Pull request merged**